docs/reference/modules/piece-page-type.md

```diff
@@ -0,0 +1,240 @@
+      {
+        name: 'category',
+        counts: true
+      }
+    ]
+  }
+};
+
+```
+  <template v-slot:caption>
+    modules/book-page/index.js
+  </template>
+</AposCodeBlock>
+
+### `pieceModuleName`
+Piece page types are associated with a single piece type. If named with the pattern `[piece name]-page`, the associated piece type will be identified automatically. You can override this pattern by explicitly setting `pieceModuleName` to an active piece type. This is useful if there is more than one piece page type for a single piece type (e.g., to support different functionality in each).
+
+<AposCodeBlock>
+
+```javascript
+// 👆 This module name would look for a piece type
+// named `fiction` if not for `pieceModuleName`
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    pieceModuleName: 'book'
+  }
+  // Code to select and group only fiction books
+  // …
+};
+```
+<template v-slot:caption>
+modules/fiction-page/index.js
+</template>
+</AposCodeBlock>
+
+### `previous`
+If this option is set to true, it exposes the previous piece in the current [sort order](https://v3.docs.apostrophecms.org/reference/module-api/module-options.html#localized) as `req.data.previous` (`data.previous` in the template) when serving a [show page](https://v3.docs.apostrophecms.org/reference/glossary.html#general-terms). This can be used to provide a link to the previous item in a series (e.g., the next newest blog post).
+
+This option can also be set to an object containing an object with a key of `project` and values which will be used as a query builder for retrieving the previous piece document.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    previous: true
+  },
+  // …
+}
+
+// OR
+
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    // The previous article piece would be returned with only the `title`, `_url`
+    // and `lastPublishedAt` properties.
+    next: {
+      project: {
+        title: 1,
+        _url: 1,
+        lastPublishedAt: 1
+      }
+    }
+  },
+  // …
+}
+```
+<template v-slot:caption>
+  modules/article-page/index.js
+</template>
+</AposCodeBlock>
+
+## Related Documentation
+
+* [Piece index and show pages](https://v3.docs.apostrophecms.org/guide/piece-pages.html)
+* [Piece page type options](https://v3.docs.apostrophecms.org/reference/module-api/module-options.html#options-for-piece-page-types)
+
+## Featured methods
+The following methods belong to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/piece-page-type/index.js) for all the modules that belong to this module.
+
+### `indexQuery(req)`
+This method should be overridden for a piece-type to call additional [query builders](https://v3.docs.apostrophecms.org/reference/query-builders.html#query-builders) by default.
```

Comment:
replace "by default" with "when generating the index page."

The base class version of the method returns a query object. Overriding looks like:

```
extendMethods(self) {
  return {
    indexQuery(_super, req) {
      // Works if there is a select field named "category"
      return _super(req).category('fiction');
    }
  };
}
```

Reply:
Should I include this example?